src/txmempool.cpp

```diff
@@ -416,6 +416,10 @@ void CTxMemPool::removeUnchecked(txiter it, MemPoolRemovalReason reason)
     for (const CTxIn& txin : it->GetTx().vin)
         mapNextTx.erase(txin.prevout);
 
+    if (mempool.m_unbroadcast_txids.erase(hash)) {
+        LogPrint(BCLog::NET, "Removed %i from m_unbroadcast_txids before GETDATA was received. \n", hash.GetHex());
```

Comment:
```suggestion
        LogPrint(BCLog::NET, "Removed %i from m_unbroadcast_txids before tx was sent out. \n", hash.GetHex());
```

Reply:
I updated it to "from m_unbroadcast_txids before confirmation that txn was sent out", but I feel uncertain... saying "before GETDATA was received" seems more clear & specific than "confirmation that txn was sent out". Could you explain further why you didn't like it? 

is this related to [this](https://github.com/bitcoin/bitcoin/pull/18038#discussion_r385795095)?